src/components/Table/components/TableRowSkeleton.tsx

```diff
@@ -15,7 +15,7 @@ export const TableRowSkeleton = ({ columns, animated }: TableRowSkeletonProps) =
                 .fill(0)
                 .map((_, index) => (
                     <TableCell key={index}>
-                        <Skeleton animated={animated} />
+                        <Skeleton aria-label="table-row-skeleton" aria-hidden animated={animated} />
```

Comment:
Should we also add a test to check that the skeleton rows are "aria-hidden"?

Reply:
Done!